src/utils/sounds.js

```diff
@@ -132,7 +142,7 @@ export const Sounds = {
 		}
 		this.lastPlayedLeave = currentTime
 
-		this._playSounceOnce('LibremTextMessage.ogg')
+		this._playSoundOnce('leave_call.wav')
```

Comment:
/apps/spreed/img/leave_call.wav:1 Failed to load resource: the server responded with a status of 404 (Not Found)
Uncaught (in promise) DOMException: Failed to load because no supported source was found.